sdcm/cluster.py

```diff
@@ -2743,8 +2743,7 @@ def _publish_event(subtype, severity, duration=None, error=None, full_traceback=
             return result
         except Exception as details:  # pylint: disable=broad-except
             event_severity_on_failure = Severity.ERROR
-            if warning_event_on_exception and type(details) in warning_event_on_exception or \
-                    Exception in warning_event_on_exception:
```

Comment:
This line should be kept, because of caught exception may be not "Exception" type. But we want to decrease event level for every kind of exception

Reply:
same as above